recipes/glibmm/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".1" %}
+{% set sha256 = "dc225f7d2f466479766332483ea78f82dc349d59399d30c00de50e5073157cdf" %}
+{% if linux %}
+    {% set libext = ".so" %}
+{% else %}
+    {% set libext = ".dylib" %}
+{% endif %}
+
+package:
+  name: glibmm
+  version: {{ version }}
+
+source:
+  fn: glibmm-{{ version }}.tar.xz
+  url: http://ftp.gnome.org/pub/GNOME/sources/glibmm/{{ major_minor }}/glibmm-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - libsigcpp
+    - glib
+    - xz
```

Comment:
We are [recommending](https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies) to pin libzma as `xz 5.0.*`. That way we avoid getting a modern version and being incompatible with Python 3.x